python/simulation.py

```diff
@@ -3201,6 +3206,9 @@ def get_dft_array(self, dft_obj, component, num_freq):
         else:
             dft_swigobj = dft_obj
 
+        if num_freq > dft_obj.nfreqs-1:
+            raise ValueError("get_dft_array: frequency index {} exceeds max index {}".format(num_freq,dft_obj.nfreqs-1))
```

Comment:
Arguably, this should be fixed in the C++ code in `process_dft_component`

Reply:
This `ValueError` exception has been removed from this PR.